etc/kayobe/kolla/config/prometheus/prometheus.yml.d/70-oscapacity.yml

```
@@ -6,4 +6,7 @@ scrape_configs:
           - '{{ kolla_internal_vip_address | put_address_in_context('url') }}:9000'
     scrape_interval: 15m
     scrape_timeout: 10m
+    {% if kolla_enable_tls_internal | bool %}
+    scheme: https
+    {% endif %}
```

Unsure if this will render properly with indented jinja conditions
```suggestion
{% if kolla_enable_tls_internal | bool %}
    scheme: https
{% endif %}
```

---

Dropped this commit as the os_capacity exporter does not expose a TLS endpoint.